src/components/inputs/StringInput.jsx

```diff
@@ -33,6 +33,7 @@ class StringInput extends React.Component {
         value: props.value
       };
     }
+    return {};
```

Comment:
`getDerivedStateFromProps` must alway return `Object`, otherwise it'll show a warning.